external-crates/move/crates/move-compiler/tests/move_2024/typing/dot_call_private.exp

```diff
@@ -1,18 +1,18 @@
 error[E04001]: restricted visibility
-   ┌─ tests/move_2024/typing/dot_call_private.move:18:5
+   ┌─ tests/move_2024/typing/dot_call_private.move:16:5
    │
- 7 │ fun f(_self: &X) {}
+ 5 │ fun f(_self: &X) {}
    │     - This function is internal to its module. Only 'public', 'public(friend)', and 'public(package)' functions can be called outside of their module
```

Comment:
Should this error message differ based on `public(friend)` being available?

Reply:
Good catch!